modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1233,6 +1233,39 @@ Use `DATABASE *` without the parameter to revoke the privilege on all databases.
 
 Topology notifications provide additional information related to managing databases and servers.
 
+[#_neo_clientnotification_cluster_cordonedserversexistedduringallocation]
+=== CordonedServersExistedDuringAllocation
+.Notification details
+[cols="<1s,<4"]
+|===
+|Code
+m|Neo.ClientNotification.Cluster.CordonedServersExistedDuringAllocation
+|Title
+a| Cordoned servers existed when making an allocation decision.
+|Severity
+m|INFORMATION
+|Category
+m|TOPOLOGY
+|===
+
+.Cordoned servers existed during an allocation decision
```

Comment:
```suggestion
.Cordoned servers exist during an allocation decision
```

Reply:
I went for past tense here as the notification is returned after the decision was made and may not be true now, i.e., if someone happened to uncordoned the server straight after you triggered an allocation decision